### ISSUE TITLE
iOS16 press on keyboard doesn't work

### ISSUE THREAD

**Issue by azilbershtein** [kif-framework/KIF#1267]
Can't press on the native keyboard on iOS16 devices, using both accessibility id and label.
Did anyone experienced that issue as well?

**Comment by dostrander** [comment]
@azilbershtein can you give a little context on how you are trying to press the native keyboard and if it's the standard keyboard or something more customized

**Comment by azilbershtein** [comment]
@dostrander Native keyboard, I've tried both ".tapView(withAccessibilityLabel: "search") and .tapView(withAccessibilityIdentifier: "Search")
Failed with "Failure in child step: The step timed out after 30.00 seconds: Failed to find accessibility element with the label "search" (KIFFailureException)"
Works perfectly with iOS15.5
![Screen Shot 2022-09-20 at 16 00 32](https://user-images.githubusercontent.com/64828042/191264416-09d80cbf-d165-419d-8e31-05005204e28e.png)
![Screen Shot 2022-09-20 at 16 01 31](https://user-images.githubusercontent.com/64828042/191264599-6c1022b5-848e-4041-96e5-366eb37c7698.png)

**Comment by dostrander** [comment]
@azilbershtein Hmmm i'll check this out, since you are seeing this in the view debugger it seems like it's inside of the process still which is good. We may be doing something in KIF to somehow skip finding this. Let me look into it

**Comment by heidiproske-qz** [comment]
We're also encountering this error. I tried to update our tests to run on an iOS 16 simulator but a number of our tests interacting with the keyboard (like the `search` button) failed. We'd previously been using iOS 15.5 simulators - so I'll stick with those in the meantime.
